_data/cryptocurrencies.yml

```diff
@@ -186,6 +186,13 @@ websites:
       software: Yes
       doc: https://support.coins.ph/hc/en-us/articles/202604924-What-is-two-factor-authentication-
 
+    - name: COINUT
+      url: https://coinut.com/
+      img: coinut.png
+      tfa: Yes
+      software: Yes
+      doc: https://help.coinut.com/hc/en-us/articles/360005061354-What-is-Two-Factor-Authentication-2FA-
```

Comment:
Please change the doc url to `https://help.coinut.com/hc/en-us/articles/360005061354`

Reply:
Chagned in the new commit.